### PR TITLE
adding pfsense temperature sensor

### PR DESCRIPTION
**Pull request by ervin09**
the following PR provides support for the pfsesne temperature. to be graphed on the GUI. 
this is what it adds into the web gui : [https://imgur.com/touBuV3](url)
Screenshot_1

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by ervin09** [comment]
hmm, i think i need a little bit help as this is dirty-PR.... only the last 2 files regarding pfsense are needed, but im currently seeing 35 commits. (these are the commits on the past for another PR, of infortrend) any ideas how i can discard them?

File: .gitignore
Comment (PipoCanaja) [inline .gitignore:1]: Could you please undo this change in the PR.

File: includes/definitions/discovery/infortrend.yaml
Comment (PipoCanaja) [inline includes/definitions/discovery/infortrend.yaml:1]: Could you please undo this change as well ?

**Comment by PipoCanaja** [comment]
Hi @ervin09 
You may reopen a new PR with only one commit instead of this one with so many commits for only 2 file changed ...

**Review by PipoCanaja — CHANGES_REQUESTED** [review]

**Comment by ervin09** [comment]
@PipoCanaja let me do another PR , this one really got out of hands. sorry.
